app/views/customer/_top_nav.html.haml

```diff
@@ -1,6 +1,6 @@
 %nav.navbar.navbar-expand-lg.navbar-light.bg-light.fixed-top
   .container
-    %a.navbar-brand{:href => "#"} Sacachi Rails Shop
+    %a.navbar-brand{:href => "/"} Sacachi Rails Shop
```

Comment:
Use named path (like in this case, "root_path") instead of absolute/relative path